e2e-tests/cypress/e2e/journeys/insurance/check-your-answers/your-business/change-your-answers/change-your-answers-change-broker.spec.js

```diff
@@ -0,0 +1,270 @@
+import { submitButton } from '../../../../../pages/shared';
+import partials from '../../../../../partials';
+import { ROUTES, FIELD_IDS } from '../../../../../../../constants';
+import { checkYourAnswersYourBusiness } from '../../../../../pages/insurance/check-your-answers';
+import { broker } from '../../../../../pages/your-business';
+import {
+  checkChangeLinkUrl,
+  changeAnswerField,
+  checkChangeAnswerRendered,
+} from '../../../../../../support/check-summary-list-field-change';
+import { EXPORTER_BUSINESS_FIELDS as FIELDS } from '../../../../../../../content-strings/fields/insurance/exporter-business';
+
+const {
+  ROOT,
+  CHECK_YOUR_ANSWERS: {
+    YOUR_BUSINESS,
+  },
+  EXPORTER_BUSINESS: {
+    BROKER_CHECK_AND_CHANGE,
+  },
+} = ROUTES.INSURANCE;
+
+const {
+  INSURANCE: {
+    EXPORTER_BUSINESS: {
+      BROKER: {
+        USING_BROKER,
+        NAME,
+        ADDRESS_LINE_1,
+        ADDRESS_LINE_2,
+        TOWN,
+        COUNTY,
+        POSTCODE,
+        EMAIL,
+      },
+    },
+  },
+} = FIELD_IDS;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.submitApplication.tasks.checkAnswersAndSubmit;
+
+const { summaryList } = checkYourAnswersYourBusiness;
+
+const getFieldVariables = (fieldId, referenceNumber) => ({
+  route: BROKER_CHECK_AND_CHANGE,
+  checkYourAnswersRoute: YOUR_BUSINESS,
+  newValueInput: '',
+  fieldId,
+  referenceNumber,
+  summaryList,
+  changeLink: summaryList.field(fieldId).changeLink,
+});
+
+context('Insurance - Check your answers - Broker - Your business - Summary list', () => {
+  let referenceNumber;
+  let url;
+
+  before(() => {
+    cy.completeSignInAndGoToApplication().then((refNumber) => {
+      referenceNumber = refNumber;
+
+      cy.completePrepareApplicationSinglePolicyType();
+
+      task.link().click();
+
+      // to get past eligibility check your answers page
+      submitButton().click();
+      // to get past policy and exports check your answers page
+      submitButton().click();
+
+      url = `${Cypress.config('baseUrl')}${ROOT}/${referenceNumber}${YOUR_BUSINESS}`;
+
+      cy.url().should('eq', url);
+    });
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+
+    cy.navigateToUrl(url);
+  });
+
+  after(() => {
+    cy.deleteAccount();
```

Comment:
`cy.deleteAccountAndApplication(referenceNumber);` for all new E2E test please, otherwise could break dashboard E2E tests